inference-engine/tests/ie_test_utils/functional_test_utils/layer_tests_summary/template/report_template.html

```diff
@@ -32,37 +32,65 @@ <h2>Operations coverage summary {{ timestamp }}</h2>
 <table class="table table-hover" id="report">
     <thead>
     <tr>
-        <th scope="col">Operation</th>
+        <th style="background-color:#F4AE6F" scope="col">Operation</th>
         {% for d in devices -%}
-        <th>{{ d }}</th>
+        <th style="background-color:#F4AE6F">{{ d }}</th>
         {% endfor %}
     </tr>
     </thead>
+    <tbody>
+        <tr>
+        <th style="background-color:#C5B8B8" scope="row">Total: {{ordered_ops|length}}</th>
```

Comment:
I would recommend to use bootstrap tables styling (e.g. `<td class="table-warning">No tests</td>`)  instead of custom colours to keep whole report in the unified color palette 
https://getbootstrap.com/docs/4.0/content/tables/#contextual-classes

Reply:
Fixed